### PR TITLE
TableToMarkdown docs

### PR DESCRIPTION
**Pull request by rshunim**
## Status
Ready

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5558)

## Description
Adding new documentation to the two new arguments and re-documenting the functions of the old arguments.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/integrations/code-conventions.md [hunk 1]
```diff
@@ -676,9 +676,36 @@ The above will create the table seen below:
 
 In the War Room, this is how a table will appear:
 <img width="788" src="/doc_imgs/integrations/50571324-46846e00-0db0-11e9-9888-ddd9dc275541.png"></img>
-
+#
 You may also use ```headerTransform``` to convert the existing keys into formatted headers.
+A function that formats the original data headers (optional).
```
Comment (ShirleyDenkberg) [inline docs/integrations/code-conventions.md:681]:
```suggestion
|This function formats the original data headers (optional).
```

File: docs/integrations/code-conventions.md [hunk 2]
```diff
@@ -711,7 +738,99 @@ tableToMarkdown('Data Table', d, headers=('id', 'url1', 'result', 'links'),
...
+                date_fields=['create_time'])
+```
+The above will create the table seen below:
+### tableToMarkdown test
+|docker_image|create_time|
+|---|---|
+| demisto/python3 | 2021-09-13 08:21:53 |
 
+#
+Use the ```json_transform_mapping``` argument (Dict[str, JsonTransformer]), mapping between a header key to corresponding JsonTransformer.
```
Comment (ShirleyDenkberg) [inline docs/integrations/code-conventions.md:760]:
```suggestion
Use the ```json_transform_mapping``` argument (Dict[str, JsonTransformer]), to map between a header key to the corresponding JsonTransformer.
```

File: docs/integrations/code-conventions.md [hunk 3]
```diff
@@ -711,7 +738,99 @@ tableToMarkdown('Data Table', d, headers=('id', 'url1', 'result', 'links'),
...
+    }
+  ]
+}
+table = tableToMarkdown("tableToMarkdown test", data_with_list,
+            json_transform_mapping={'Commands': JsonTransformer(keys=('commandStatus', 'command'))})
+```
+The above will create the table seen below:
+![img_1.png](img_1.png)
+#
+Use the ```is_auto_json_transform``` argument (bool), to try to auto transform complex json.
```
Comment (ShirleyDenkberg) [inline docs/integrations/code-conventions.md:807]:
```suggestion
Use the ```is_auto_json_transform``` argument (bool), to try to auto transform a complex JSON.
```

**Comment by ShirleyDenkberg** [comment]
@dorschw Doc review completed.

File: docs/integrations/code-conventions.md [hunk 4]
```diff
@@ -676,9 +676,36 @@ The above will create the table seen below:
...
+#
 You may also use ```headerTransform``` to convert the existing keys into formatted headers.
+|This function formats the original data headers (optional).
+
+```python
+t = {'header_1': 'a1', 'header_2': 'b1', 'header_3': 'c1'}
+tableToMarkdown('tableToMarkdown with headerTransform', t, headerTransform=underscoreToCamelCase)
+```
+The above will create the table seen below:
+### tableToMarkdown with headerTransform
```
Comment (dorschw) [inline docs/integrations/code-conventions.md:688]:
```suggestion
tableToMarkdown('headerTransform Example', t, headerTransform=underscoreToCamelCase)
```
The above will create the table seen below:
### headerTransform Example
```

File: docs/integrations/code-conventions.md [hunk 5]
```diff
@@ -676,9 +676,36 @@ The above will create the table seen below:
...
+You may also use ```removeNull``` to remove empty columns in the table. Default is False.
+```python
+headers = ['header_1', 'header_2']
+data = {
+    'header_1': 'foo',
+}
+tableToMarkdown('tableToMarkdown test', data, removeNull=True, headers=headers)
+```
+The above will create the table seen below:
+### tableToMarkdown test
```
Comment (dorschw) [inline docs/integrations/code-conventions.md:702]:
```suggestion
tableToMarkdown('removeNull Example', data, removeNull=True, headers=headers)
```
The above will create the table seen below:
### removeNull Example
```

File: docs/integrations/code-conventions.md [hunk 6]
```diff
@@ -711,7 +738,99 @@ tableToMarkdown('Data Table', d, headers=('id', 'url1', 'result', 'links'),
...
+        "docker_image": "demisto/python3",
+        "create_time": '1631521313466'
+    }
+]
+tableToMarkdown('tableToMarkdown test', data, headers=["docker_image", "create_time"],
+                date_fields=['create_time'])
+```
+The above will create the table seen below:
+### tableToMarkdown test
+|docker_image|create_time|
```
Comment (dorschw) [inline docs/integrations/code-conventions.md:755]:
```suggestion
tableToMarkdown('tableToMarkdown date_fields example', data, headers=["docker_image", "create_time"],
                date_fields=['create_time'])
```
The above will create the table seen below:
### tableToMarkdown date_fields example
|docker_image|create_time|
```

**Review by dorschw — APPROVED** [review]

**Pull request merged**